docs/book/src/developer/testing.md

```diff
@@ -158,12 +158,12 @@ kind images). This can be done by executing the `./scripts/ci-e2e.sh` script.
 
 ```bash
 # Notes:
-# * You can cancel the script as soon as it starts the actual test execution via `make -C test/e2e/ run`.
+# * You can cancel the script as soon as it starts the actual test execution via `make run`.
 # * If you want to run other tests (e.g. upgrade tests), make sure all required env variables are set (see the Prow Job config).
 GINKGO_FOCUS="\[PR-Blocking\]" ./scripts/ci-e2e.sh
 
 # Make sure the cluster-templates have been generated.
-make -C test/e2e cluster-templates
+make cluster-templates
```

Comment:
let's move the call to make templates before make run

Reply:
I fully removed because it's a prerequisite for the run-e2e target. So the user shouldn't normally need to explicitly call it.